app/src/main/res/values/attrs.xml

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <declare-styleable name="HintLabelEditText">
-        <attr name="hint" format="string" />
-        <attr name="label" format="string" />
+        <attr name="hint" format="string"/>
+        <attr name="label" format="string"/>
+        <attr name="android:inputType"/>
     </declare-styleable>
     <declare-styleable name="ProgressIndicatorTextView">
```

Comment:
I think I should add an extra line between these two declare-styleable blocks @jmccormack200